deployment/clusterObjectModel/test/data/configuration-template-generate/layout.yaml

```diff
@@ -19,7 +19,7 @@
 #TODO
 kubernetes:
   api-servers-url: http://10.0.0.1:8080
-  dashboard-host: 10.0.0.1
```

Comment:
dashboard-ip-address?

Reply:
dashboard-url already used in current layout.yaml. Please refer to https://github.com/microsoft/pai/blob/3adbcddada8fa95844e065843f4ed3009edac967/deployment/quick-start/layout.yaml.template#L23-L24

I'm not sure why there is a field call `dashboard-host`, this filed is not documented